app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -62,6 +63,7 @@ open class DataStore(
                         is DataStoreAction.Lock -> lock()
                         is DataStoreAction.Unlock -> unlock()
                         is DataStoreAction.Sync -> sync()
+                        is DataStoreAction.Reset -> reset()
```

Comment:
Reset Action is only for test purpose? Did not see it dispatched from anywhere else beside test.